### PR TITLE
Add page1170.txt

### PR DESCRIPTION
**Pull request by sebras**

### CODE REVIEW

File: transcribed/page1170.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+001                                                                             PAGE 117
+002
+003     003 004 SUBTTL RANDOM STORAGE
+004
+005             RADIO 10.
```
Comment (larsbrinkhoff) [inline transcribed/page1170.txt:5]: RADIX

File: transcribed/page1170.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+006     002 007 REPEAT SIOML,[CONC TRAN,\.RRPCNT+1,:
```
Comment (larsbrinkhoff) [inline transcribed/page1170.txt:6]: Missing BLOCK SIOMT

File: transcribed/page1170.txt [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+024             SCRFLG: 0       ;CURRENT USER MUST BE INTERRUPTED, SAVE VARIABLES, MUNG AND RESTORE
+025             SCRFG1: 0       ;LOSE LOSE LOSE LOSE LOSE LOSE LOSE LOSE LOSE LOSE LOSE LOSE
+026             UTTYST: -1      ;-1 OF TTY THAT WANTS TO BE STARTED
+027             CRACF:  -1      ;CORE ALLOCATOR FLAG (ONLY 1 AT A TIME)
+028     002 007 SIOMTU: SIOMT   ;AMOUNT USED OF I-O MAPPING TABLE
+029             SLCLK:  0       ;SLOW CLOCK
+030             SRCS:   0       ;RECYCLE FLAG FOR SEARCH
+031
+032             CONSTANTS
+033             VARIABLE
```
Comment (larsbrinkhoff) [inline transcribed/page1170.txt:33]: VARIABLES

File: transcribed/page1170.txt [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+034             SSYB==1+._-10.+IFN .&1777,1
```
Comment (larsbrinkhoff) [inline transcribed/page1170.txt:34]: SYSB

**Review by larsbrinkhoff — CHANGES_REQUESTED** [review]

File: transcribed/page1170.txt (hunk 1 above)
Comment (sebras) [reply transcribed/page1170.txt:5]: Nope! I like the radio better! But you're right of course... :)

File: transcribed/page1170.txt (hunk 2 above)
Comment (sebras) [reply transcribed/page1170.txt:6]: Ooops.

File: transcribed/page1170.txt (hunk 3 above)
Comment (sebras) [reply transcribed/page1170.txt:33]: Tes.

File: transcribed/page1170.txt (hunk 4 above)
Comment (sebras) [reply transcribed/page1170.txt:34]: Agreed.

**Pull request merged**